modules/rest/src/main/java/org/apache/ignite/internal/rest/cluster/ClusterManagementController.java

```diff
@@ -86,7 +88,14 @@ public CompletableFuture<Void> init(@Body InitCommand initCommand) {
         });
     }
 
-    private static ClusterState mapClusterState(org.apache.ignite.internal.cluster.management.ClusterState clusterState) {
+    private static ClusterState mapClusterState(@Nullable org.apache.ignite.internal.cluster.management.ClusterState clusterState) {
+        if (clusterState == null) {
+            throw new IgniteException(
+                    INTERNAL_ERR,
```

Comment:
Ideally, it should not be an internal error. Maybe we should introduce a new exception and code?

Reply:
I think an exception is not necessary here, I added the error code.